frontend/app/src/components/global-components/navbar.tsx

```diff
@@ -182,7 +184,7 @@ function Navbar({ text, avatarImg }: BannerProps) {
             </DropDownMenu>
```

Comment:
Remove Profile from dropDownMenu in navbar

Reply:
ahh je l'aurais bien gardé en plus celui-ci